src/GslCore/PluginTypes.fs

```diff
@@ -218,13 +243,15 @@ let configureBehavior arg b =
     | AssemblyTransform(a) -> {b with behavior = AssemblyTransform(a.Configure(arg))}
     | CodonProvider(c) -> {b with behavior = CodonProvider(c.Configure(arg))}
     | AlleleSwapAA _
+    | MarkerProvider _

```

Comment:
Since MarkerProvider inherits from IConfigurable, it should be included in the behaviors that are configured at this step or they will never be passed their own command line arguments.